app/config/AppConfig.scala

```diff
@@ -55,6 +55,7 @@ class AppConfig @Inject()(
   lazy val loginUrl = loadConfig("urls.login")
   lazy val loginContinueUrl = loadConfig("urls.loginContinue")
   lazy val customsDeclarationsGoodsTakenOutOfEuUrl = loadConfig("urls.customsDeclarationsGoodsTakenOutOfEu")
+  lazy val tradeTariffUrl = loadConfig("urls.tradeTariff")
```

Comment:
I think that you're not using this.

You have this created in `MucrOptionsConfig.scala`

Reply:
MucrOptionsConfig uses AppConfig

Reply:
Ah, yeah, I though that we're retrieving this from config just in MucrOptionsConfig